cmd/spec/path.yaml

```diff
@@ -148,9 +146,9 @@ paths:
           content:
             application/json:
               schema:
-                type: array
+                type: 
                 items:
```

Comment:
I guess we forgot it here @adiabramovitch ? :)

Reply:
Sorry! fixed it